assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/index.js

```diff
@@ -41,72 +45,34 @@ import {
 	ACTIVATION_STEP_SETUP,
 	ACTIVATION_STEP_SUCCESS,
 	ENHANCED_MEASUREMENT_ACTIVATION_BANNER_TOOLTIP_STATE_KEY,
-	ENHANCED_MEASUREMENT_ACTIVATION_BANNER_DISMISSED_ITEM_KEY,
 } from '../../../constants';
 import { useTooltipState } from '../../../../../components/AdminMenuTooltip/useTooltipState';
 import { useShowTooltip } from '../../../../../components/AdminMenuTooltip/useShowTooltip';
 import { AdminMenuTooltip } from '../../../../../components/AdminMenuTooltip/AdminMenuTooltip';
 import InProgressBanner from './InProgressBanner';
 import SetupBanner from './SetupBanner';
 import SuccessBanner from './SuccessBanner';
-import { MONTH_IN_SECONDS, trackEvent } from '../../../../../util';
-import whenActive from '../../../../../util/when-active';
-import {
-	isValidPropertyID,
-	isValidWebDataStreamID,
-} from '../../../utils/validation';
-import useViewContext from '../../../../../hooks/useViewContext';
-
-function EnhancedMeasurementActivationBanner() {
-	const viewContext = useViewContext();
 
+export default function EnhancedMeasurementActivationBanner( {
+	id,
```

Comment:
Similar to my previous comment: let's not add anymore standard HTML attributes as props that override the "expected" functionality of said HTML attributes. Things like `id` or `role` are best not to be entirely overridden as it can get confusing.